### DISCUSSION TITLE
点击卡片的刷新按钮,再打开新的 tab, 还会提示需要刷新, 这大概是什么原因呢?

### DISCUSSION THREAD

**Discussion by vaynevayne** [GreatAuk/plugin-web-update-notification#45] [Q&A]
如题

**Comment by GreatAuk** [comment]
有缓存，建议把 `index.html` 的缓存禁了
https://github.com/GreatAuk/plugin-web-update-notification/blob/master/README.zh-CN.md#%E5%BB%BA%E8%AE%AE-%E5%A6%82%E6%9E%9C%E6%98%AF-spa-%E5%BA%94%E7%94%A8%E7%A6%81%E7%94%A8-indexhtml-%E7%9A%84%E7%BC%93%E5%AD%98
